trikot-viewmodels-declarative/swift/swiftui/Utilities/ObservableViewModelAdapter.swift

```diff
@@ -7,13 +7,19 @@ public class ObservableViewModelAdapter<VM: VMDViewModel>: ObservableObject {
     private let propertyWillChangePublisher: AnyPublisher<KotlinUnit, Never>
     private var cancellable: AnyCancellable?
 
-    public init(viewModel: VM) {
+    public init(viewModel: VM, animation: Animation? = nil) {
```

Comment:
I don't think that globally set animations for any property change is a good idea. We should be able to decide, for a given property change if we want that change to be animated or not. Animating everything would cause more harm than good IMO.